### PR TITLE
Update export-to-csv.md to 3.0

### PR DESCRIPTION
**Pull request by jmarsh24**

### CODE REVIEW

**Comment by adrianthedev** [comment]
I would like to keep the fields to just as an example of how that can be achieved.

**Comment by jmarsh24** [comment]
Is there a way to not require the user to select the left box -> select all records. What if a user wants to just download the entire table and always have this button available?

it's also not clear if the check_box attributes should be separated if this csv_export is used on different models for instance. It could be good to add an example for that in terms of **best practice**.

**Comment by adrianthedev** [comment]
1. yes. https://docs.avohq.io/3.0/actions.html#standalone-actions
2. I didn't understand that. Could you explain a bit?

File: docs/3.0/recipes/export-to-csv.md [hunk 1]
```diff
@@ -36,10 +36,13 @@ class ExportCsv < Avo::BaseAction
     download file, "#{resource.plural_name}.csv"
   end
 
-  def get_columns_from_fields(fields)
-    fields.select { |key, value| value }.keys
+  def get_attributes(record)
+    # return ["id", "created_at"] # uncomment this and fill in for custom model properties
+
+    record.class.columns_hash.keys
```
Comment (adrianthedev) [inline docs/3.0/recipes/export-to-csv.md:42]:
```suggestion
    # return record.class.columns_hash.keys # uncomment to get all the models' attributes.
    
    # get the attributes that the user selected.
    fields.select { |key, value| value }.keys
```

**Review by adrianthedev — APPROVED** [review]

**Comment by jmarsh24** [comment]
@adrianthedev 
The intuitive default I think should be to export the entire table if nothing is selected. This is what I'm trying to achieve below but it's not straightforward on how to achieve this from the documentation. This is where I would say that the documentation can be improved here.

image

```ruby
class Avo::Actions::ExportCsv < Avo::BaseAction
  self.name = "Export csv"
  self.no_confirmation = true
  self.standalone = true

  def handle(**args)
    records, resource = args.values_at(:records, :resource)
    records = resource.all if records.empty?

    attributes = get_attributes records.first

    file = CSV.generate(headers: true) do |csv|
      csv << attributes

      records.each do |record|
        csv << attributes.map do |attr|
          record.send(attr)
        end
      end
    end

    download file, "#{resource.plural_name}_#{Time.zone.now.strftime("%Y%m%d%H%M%S")}.csv"
  end

  def get_attributes(record)
    # return ["id", "created_at"] # uncomment this and fill in for custom model properties

    record.class.columns_hash.keys
  end
end
```

**Comment by adrianthedev** [comment]
I updated the example to support that. 
By making it `standalone` you are enabling people to use the app when no record is selected.
I added code to uncomment to support multiple use cases. Tested too.

Thanks @jmarsh24

**Pull request merged**